utils/sql-db-seeder/src/index.ts

```diff
@@ -40,3 +51,6 @@ const run = async (): Promise<void> => {
 (async () => {
   await run();
 })();
+run()
+  .then(() => {})
+  .catch(() => {});
```

Comment:
Why do we need empty then's and catches?

Looks like `run()` will be exectued twice

Reply:
This was a mistake - will remove